opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipOverview.jsx

```diff
@@ -241,13 +241,31 @@ class StixCoreRelationshipContainer extends Component {
           classes={{ container: classes.gridContainer }}
         >
           <Grid item xs={6}>
-            <Typography variant="h4" gutterBottom={true}>
+            <Typography variant="h4" gutterBottom={false} sx={{ float: 'left' }}>
               {t('Relationship')}{stixCoreRelationship.draftVersion && (<DraftChip/>)}
             </Typography>
+            {!stixCoreRelationship.is_inferred && (
+            <Security needs={[KNOWLEDGE_KNUPDATE]}>
+              <IconButton
+                color="primary"
+                onClick={this.handleOpenEdition.bind(this)}
+                size="large"
+                sx={{ margin: '-15px 0px 0px -2px' }}
+              >
+                <EditOutlined fontSize="small" />
+              </IconButton>
+              <StixCoreRelationshipEdition
+                open={this.state.openEdit}
+                stixCoreRelationshipId={stixCoreRelationship.id}
+                handleClose={this.handleCloseEdition.bind(this)}
+                handleDelete={this.handleOpenDelete.bind(this)}
+              />
+            </Security>
+            )}
             <Paper
               classes={{ root: classes.paperWithoutPadding }}
               variant="outlined"
-              style={{ position: 'relative' }}
+              style={{ position: 'relative', marginTop: '-4px' }}
```

Comment:
Ugly because there is somewhere another inline style overriding every `sx` props we put on the component.